### ISSUE TITLE
General usage question - Public vs Private setup

### ISSUE THREAD

**Issue by ozh** [YOURLS/YOURLS#214]
This is a **COPY** of **Issue 214: [General usage question - Public vs Private setup](http://code.google.com/p/yourls/issues/detail?id=214)**, filed on Google Code before the project was [moved on Github](https://github.com/ozh/google-issues-to-github).
- Submitted on 2010-01-05T22:18:20.000Z by [kennedy...@gmail.com](http://code.google.com/u/118120124922535876861/)
- Status: Invalid

Please **review** the original issue and especially its comments. **Comments here on closed issues will be ignored**. Thanks.
## Original description

```
I have some general usage questions about Yourls.  Is there a better place
to post them than here?

Specifically, I'm a bit confused about the differences between a public and
private set up, and the fact that even if you set it up as private, you can
still have a public facing page.

Basically, this is what I'd like:
Anyone can create short URLs
Creators can see stats and reports
No one can delete someone else's URLs

When Yourls is set to public, is there a way to prevent someone from just
deleting all the URLs in the admin panel?

Or if it's set to private, and URLs are created via the public page, is
there a way to allow creators to get stats for their URLs?  Should I just
give out logins and passwords by request?

Thanks
```
